### PR TITLE
PD-2382 - Added function to scroll report to top, when a report is run.

### PR DESCRIPTION
**Pull request by misterjonscott**
# It's nothing complex.
I added a loop to scroll over 250ms so the user sees they're being taken back to the top, and not to a new page.

### CODE REVIEW

**Comment by aspidites** [comment]
Would it be handy to have this in common so we can use it elsewhere? I suspect User Management would be a good candidate for this next.

File: gulp/src/reporting/SetUpReport.jsx [hunk 1]
```diff
@@ -88,6 +88,15 @@ export default class SetUpReport extends Component {
     }
   }
 
+  scrollToTop() {
```
Comment (tdphillips) [inline gulp/src/reporting/SetUpReport.jsx]: Should we call something "scrollToTop" if it doesn't scroll to the top?

Comment (aspidites) [inline gulp/src/reporting/SetUpReport.jsx]: May be nice to parameterize this by the amount to scroll on each interval as well as the point at which to stop.

**Comment by misterjonscott** [comment]
@aspidites This could certainly be expanded to take inputs and be re-used on other pages, but in the interest of time, and the context of the ticket, this seems appropriate for now.  If anyone feels we should make this into a full-featured function in the common tools, I'll close this request and focus on those changes.

File: gulp/src/reporting/SetUpReport.jsx (hunk 1 above)
Comment (misterjonscott) [inline gulp/src/reporting/SetUpReport.jsx]: @tdphillips I changed the "top" back to 0, assuming that's what you meant with your comment.  Thanks for catching that.

**Pull request merged**